src/styles/blog.module.css

```diff
@@ -115,6 +115,10 @@
   margin-left: 8px;
 }
 
+.post :global(.underline) {
+  border-bottom: 1px solid black;
```

Comment:
```suggestion
  text-decoration: underline;
```

Should we use underline styling here instead?

Reply:
Thank you for review😄
fixed to text-decoration.